### DISCUSSION TITLE
REPLICATION_MODE="secondary" Host '127.0.0.1' is not listening on port '5432'

### DISCUSSION THREAD

**Discussion by icueofraj** [tiredofit/docker-postgres#16] [Q&A]
<img width="856" alt="Screenshot 2023-10-05 at 10 32 13" src="https://github.com/tiredofit/docker-postgres/assets/126460358/163b9107-fea1-43a3-82a1-d176a8cd86b6">  Host '127.0.0.1' is not listening on port '5432'

EVN
CREATE_DB=TRUE
DB_NAME="maindb"
DB_USER="master"
DB_PASS="123456"

ENABLE_REPLICATION=true
REPLICATION_MODE="secondary"
REPLICATION_USER="replicate"
REPLICATION_PASS="postgres"
REPLICATION_HOST="xxxxxxx"
REPLICATION_PORT=5432

**Comment by joergmschulz** [comment]
... so there currently only is the older variant on the docker hub which is good because it doesn't fail ...

**Comment by tctlrd** [comment]
I am getting this error on version 17 but not 18. I need 17.
